New package: x64dbg.x64dbg version snapshot_2023-10-05_13-38

Dragon1573 opened the pull request:
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/122919

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122930)

Dragon1573 committed:
New package: x64dbg.x64dbg version snapshot_2023-10-05_13-38

Dragon1573 commented:
During test, it seems that `x64dbg.exe`, `x32dbg.exe` and `x96dbg.exe` are unable to launch with command in terminal. Go to `%LOCALAPPDATA%\Microsoft\WinGet\Links` and double-click the `.symlink` icon can launch the GUI of target application.

Trenly commented on manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.installer.yaml:
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38
```
```suggestion
PackageVersion: 2023.10.05
```
The version should be either a semantic version or a date in order for Winget to sort the newest releases properly

Trenly commented on manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.locale.en-US.yaml:
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38
```
```suggestion
PackageVersion: 2023.10.05
```

Trenly commented on manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.yaml:
```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38
```
```suggestion
PackageVersion: 2023.10.05
```

Trenly commented on manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.locale.en-US.yaml:
```diff
@@ -0,0 +1,21 @@
+ShortDescription: An open-source x64/x32 debugger for windows.
+Moniker: x64dbg
+ReleaseNotesUrl: https://github.com/x64dbg/x64dbg/releases/tag/snapshot
+InstallationNotes: |
```
```suggestion
InstallationNotes: |-
```

Dragon1573 committed:
Fix: Package version

- The version should be either a semantic version or a date in order for Winget to sort the newest releases properly.

Co-authored-by: Trenly <trenlymc@gmail.com>
Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 committed:
Fix: YAML syntax

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

Dragon1573 committed:
Fix: Installation notes

The links directory may not always be the same since it is user configurable,
and so should not be referenced in strings.

Co-authored-by: Trenly <trenlymc@gmail.com>
Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>
